### PR TITLE
End of Year: Implements the Top Podcasts list style

### PR DESCRIPTION
**Pull request by emilylaguna**
| 📘 Project: #374 | 🛫 Depends on: #562 |
|:---:|:---:|

This addresses the following items:
- Updates the font sizes of all the labels
- Fixes the font size for the numbers
- Adds padding between the title and artist labels
- Fixes font weights
- Adjusts the spacings to be more inline with the designs
- Fixes the white corner appearance

This also includes the following improvements:
- The image size is now dynamically calculated based on the height of the screen which allows it to appear more consistently on smaller and larger views
- The amount of text allowed for the title and artist label is based on the height of the row which means larger screens will see more text

### Design vs Implementation Comparison

<p align="center"></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
||||

## To test

1. Launch the app
2. Tap on Profile > End of Year Card
3. Switch to the Top Podcasts story
4. ✅ Verify it looks good and matches the designs
5. ✅ Share the image and verify all the assets appear (except the logo, will be fixed)
6. Repeat the checks on various device sizes

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by pocketcasts** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
ListeningTimeStory.swift | 17 | Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: podcasts/End of Year/Stories/Views/PodcastCover.swift [hunk 1]
```diff
@@ -95,10 +96,10 @@ struct NormalCoverShadow: ViewModifier {
     func body(content: Content) -> some View {
         content
             .cornerRadius(4)
-            .shadow(color: .black.opacity(0.1 + 0.2), radius: 1, x: 0, y: 1)
-            .shadow(color: .black.opacity(0.09 + 0.2), radius: 3, x: 0, y: 3)
-            .shadow(color: .black.opacity(0.05 + 0.2), radius: 4, x: 0, y: 6)
-            .shadow(color: .black.opacity(0.01 + 0.2), radius: 4, x: 0, y: 11)
+            .shadow(color: .black.opacity(0.1), radius: 1, x: 0, y: 1)
```
Comment (emilylaguna) [inline podcasts/End of Year/Stories/Views/PodcastCover.swift]: I made this change earlier but I didn't merge that branch in yet. But I picked the change into this

File: podcasts/End of Year/Stories/Views/PodcastCover.swift [hunk 2]
```diff
@@ -43,7 +44,7 @@ struct PodcastCover: View {
                 }
             }
             .opacity(image != nil ? 1 : 0.2)
-
+            .blendMode(!renderForSharing && image != nil ? .multiply : .normal)
```
Comment (emilylaguna) [inline podcasts/End of Year/Stories/Views/PodcastCover.swift:47]: .multiply leaves the shadow, but removes the white so you end up with a "clear" shadow. But this doesn't render in screenshots so we remove it.

**Review by leandroalonso — APPROVED** [review]
Shadow is now fixed :D

**Pull request merged**